modules/oxide/src/less/theme/components/form/layout.less

```diff
@@ -49,9 +49,9 @@
     justify-content: space-between;
   }
 
+  // Escape the calc expression otherwise less incorrectly converts this to 42%
```

Comment:
Moved this, as the stylelint update started to fail with this message:
```
src/less/theme/components/form/layout.less
 55:5  ✖  Expected empty line before declaration   declaration-empty-line-before
```

Reply:
Have we logged this upstream to less?

Reply:
I'm honestly not sure if it's a bug or not... so no haven't logged an issue and just assuming it's expected for now.